plugins/available/alias-completion.plugin.bash

```diff
@@ -81,15 +86,18 @@ function alias_completion {
                         (( COMP_POINT += \${#COMP_LINE} ))
                         $compl_func \"$alias_cmd\" \"\$compl_word\" \"\$prec_word\"
                     }" >> "$tmp_file"
-                    new_completion="${new_completion/ -F $compl_func / -F $compl_wrapper }"
-            fi
-        fi
+				new_completion="${new_completion/ -F $compl_func / -F $compl_wrapper }"
+			fi
+		fi
 
-        # replace completion trigger by alias
-        if [[ -n $new_completion ]]; then
-            new_completion="${new_completion% *} $alias_name"
-            echo "$new_completion" >> "$tmp_file"
-        fi
-    done < <(alias -p | sed -Ene "s/$alias_regex/\2 '\3' '\4'/p")
-    source "$tmp_file" && command rm -f "$tmp_file"
-}; alias_completion
+		# replace completion trigger by alias
+		if [[ -n $new_completion ]]; then
+			new_completion="${new_completion% *} $alias_name"
+			echo "$new_completion" >> "$tmp_file"
+		fi
+	done < <(alias -p | sed -Ene "s/$alias_regex/\2 '\3' '\4'/p")
+	# shellcheck disable=SC1090
+	source "$tmp_file" && command rm -f "$tmp_file"
```

Comment:
[SC1090](https://github.com/koalaman/shellcheck/wiki/SC1090) mentions:

> If you don't care that ShellCheck is unable to account for the file, specify `# shellcheck source=/dev/null`.

I don't know if there's a big difference between the two options, but `source=/dev/null` seems a bit more explicit.

Reply:
I agree, `source=/dev/null` is more explicit. I'll do the change.